public/app/mu-plugins/moj-auth/401.php

```diff
@@ -72,17 +77,20 @@ public function handle401Request(): void
         // Set loginAttempts with a default of 1, or add one to the existing value.
         $jwt->login_attempts = empty($jwt->login_attempts) ? 1 : ((int) $jwt->login_attempts) + 1;
 
-        // Where to redirect the user after successful login.
-        $jwt->success_uri = $_SERVER['REQUEST_URI'];
+        // Always add the schema and domain here, to prevent an open redirect vulnerability.
+        $jwt->success_url = $_ENV['WP_HOME'] . $_SERVER['REQUEST_URI'];
```

Comment:
@wilson1000 I updated this as our previous solution used a WP function, get_home, which isn't available to us here.

See also the above check for `if(empty($_ENV['WP_HOME']))`